packages/ckeditor5-list/src/utils.js

```diff
@@ -365,6 +375,40 @@ export function getSiblingNodes( position, direction ) {
 	return items;
 }
 
+/**
+ * Returns an array with all `listItem` elements that represents the currently selected list.
+ *
+ * It returns all the items even if only part of the list is selected.
+ * If no list is selected, returns an empty array.
+ * The order of elements is not specified.
+ *
```

Comment:
If I understand correctly, this will return list items regardless of their nesting level (`listIndent`). I think this fact is worth mentioning here because somebody may think this returns list items on the same (flat) level of indentation only.